mobile/ebitenmobileview/input.go

```diff
@@ -46,4 +46,5 @@ func updateInput() {
 	}
 
 	ui.Get().UpdateInput(keys, runes, touchSlice)
+	runes = nil
```

Comment:
Why is this needed?

Reply:
This affects the Android side. Please do not this (if needed, please make another PR)

Reply:
It's probably an existing bug, but I have no code that uses the runes feature. Moved it to iOS-only now.